serverlessworkflow/modules/ROOT/pages/cloud/operator/referencing-resource-files.adoc

```diff
@@ -55,7 +64,7 @@ spec:
     end: true
 ----
 
-<1> The workflow defines an input schema
+<1> The workflow defines an input schema  
```

Comment:
is it okay to have trailing white spaces?
there are couple of places at the end of the document.

Reply:
Yes, that's fine and expected. I ran a formatter on my IDE.